rpm/fits/package.spec

```diff
@@ -44,7 +45,7 @@ mkdir -p \
 
 cp fits.sh fits-ngserver.sh fits-env.sh  %{buildroot}/usr/bin/
 cp -rf lib/* %{buildroot}/usr/share/fits/lib/
-cp lib-fits/fits.jar %{buildroot}/usr/share/fits/lib/
+cp lib-fits/fits-%{version}.jar %{buildroot}/usr/share/fits/lib/
```

Comment:
Missing a "systemctl daemon-reload" for the postscript section. After upgrading the package from 0.10.1 version I got the message:

```
[centos@mamedin-centos ~]$ sudo service fits-nailgun status
Redirecting to /bin/systemctl status fits-nailgun.service
● fits-nailgun.service - FITS Nailgun server
   Loaded: loaded (/usr/lib/systemd/system/fits-nailgun.service; enabled; vendor preset: disabled)
   Active: active (running) since Fri 2018-10-12 18:17:11 CEST; 3s ago
 Main PID: 26038 (java)
   CGroup: /system.slice/fits-nailgun.service
           └─26038 java -classpath :/usr/share/fits//lib/AES31.jar:/usr/share/fits//lib/commons-cli-1.1.jar:/usr/share/fits//lib/commons-...

Oct 12 18:17:11 mamedin-centos systemd[1]: Started FITS Nailgun server.
Oct 12 18:17:11 mamedin-centos systemd[1]: Starting FITS Nailgun server...
Oct 12 18:17:11 mamedin-centos fits-ngserver.sh[26038]: You may now run FITS by typing: ng edu.harvard.hul.ois.fits.Fits [options]
Oct 12 18:17:11 mamedin-centos fits-ngserver.sh[26038]: NGServer 0.9.2-SNAPSHOT started on all addresses, port 2113.
Warning: fits-nailgun.service changed on disk. Run 'systemctl daemon-reload' to reload units.
```

Reply:
Both changes have been addresed